### PR TITLE
Improvments/dsbd 45

### PR DESCRIPTION
**Pull request by n3op2**
### What
- Even more simplified version of existing components.
- README.md update - will expand when we have a bit more coverage

### CODE REVIEW

**Review by nitro-marky — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Morello UI
+A GUI for executing binaries on morello board
+
```
Comment (dblane-digicatapult) [inline README.md:3]: How to build?
How to start?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@digicatapult/morello-ui",
-  "version": "0.1.1",
+  "version": "0.1.2",
   "description": "User interface for interacting with morello-api",
   "main": "src/index.js",
   "scripts": {
```
Comment (dblane-digicatapult) [inline package.json:6]: Follow the convention for scripts as in other repos.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]
Please don't make structural changes without asking people first.

**Review by dblane-digicatapult — COMMENTED** [review]
Add Browserlist to package.json

```
"browserslist": {
    "production": [
      ">0.2%",
      "not dead",
      "not op_mini all"
    ],
    "development": [
      "last 1 chrome version",
      "last 1 firefox version",
      "last 1 safari version"
    ]
  }
```

File: README.md (hunk 1 above)
Comment (n3op2) [reply README.md:3]: It has been included now

File: package.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-    "lint": "eslint .",
-    "start": "webpack-dev-server --mode=development",
+    "lint": "eslint --ext .js .",
+    "start": "webpack-dev-server --mode=production",
```
Comment (dblane-digicatapult) [inline package.json:9]:
```suggestion
    "dev": "webpack-dev-server --mode=development",
    "start": "webpack-dev-server --mode=production",
```

File: package.json [hunk 4]
```diff
@@ -19,6 +19,18 @@
...
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
   "keywords": [
     "OpenAPI"
```
Comment (dblane-digicatapult) [inline package.json:36]: Remove

File: package.json (hunk 2 above)
Comment (n3op2) [reply package.json:6]: Just copied most of them. I did not use react-script as they do have some vulnarabilities and install loads of stuff we won't even need so just did simple webpack config with loaders and etc. It has been updated

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+### Local development
+Pretty much a standard one as for every other ReactJS app. There will be some .env that will require to be added/updated, but for time being it's to early.
+
+1. Start with installing dependencies
+```
+npm i
+```
+2. Start the development webpack
+```
+npm start
```
[the remark this answers is not in the document]
Comment (n3op2) [reply README.md:19]: I think we have covered this in the chat. Webpack runs as a service for local dev and then we use it to bundle stuff together and spit out lightweight js files

File: package.json (hunk 4 above)
Comment (n3op2) [reply package.json:37]: Removed!

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (dblane-digicatapult) [reply README.md:19]: No, this is for local development, thus the run command should be `npm run dev`

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**